src/System.Reflection.Metadata/src/System/Reflection/Metadata/Internal/Tables.cs

```diff
@@ -1787,23 +1794,22 @@ int containingBlockOffset
             }
         }
 
-        internal int GetRVA(uint rowId)
+        internal int GetRVA(int rowId)
```

Comment:
You've decided to go with Rva casing for consistency below. I second that decision and so you should probably do it here too.